inspect/api.html

```diff
@@ -5,172 +5,328 @@
   <meta charset="utf-8">
   <title>Grist Plugin API</title>
   <script src="https://docs.getgrist.com/grist-plugin-api.js"></script>
+  <link rel="stylesheet" data-name="vs/editor/editor.main" href="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.26.1/min/vs/editor/editor.main.min.css">
+  <script src="https://cdn.jsdelivr.net/npm/json-formatter-js@2.3.4/dist/json-formatter.umd.min.js"></script>
+  <script>
+    var require = {paths: {'vs': 'https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.26.1/min/vs'}};
+  </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.26.1/min/vs/loader.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.26.1/min/vs/editor/editor.main.nls.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.26.1/min/vs/editor/editor.main.js"></script>
+  <script src="api_deps.js"></script>
+  <script src="api.js"></script>
   <style>
-    textarea {
-      display: none;
+    h2 {
+      font-size: large;
+      font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+      color: #333;
+      padding-top: 14px;
+      margin-top: 3em;
+      border-top: 1px solid lightgray;
+    }
+    h2:first-of-type {
+      border: 0px;
+      padding: 0px;
+      margin-top: 0px;
+    }
+    .code-editor {
+      padding: 8px;
+      border: 1px solid lightgray;
+      border-radius: 2px;
+      margin-bottom: 10px;
+    }
+    .doc {
+      margin: 6px 0px;
+    }
+    body {
+      padding-bottom: 4em;
     }
   </style>
 </head>
 
 <body>
 
+  <div style="font-weight: 400;">
+    <br />
+    <i style="color:red">*Note*</i> Do not past code that you don't trust or is yours.
+  </div>   
+  <h4>
+    Feel free to explore the API and explore the API yourself. Each snippet is fully editable. <br />
+    <a target="_blank" href="https://support.getgrist.com/widget-custom#adding-a-widget">Learn more about the API</a><br />
+  </h4>
   <h2>grist.ready()</h2>
-  <script>
-    async function gristReady(cmd) {
-      cmd.output("(already called, just needs to be called once)");
+  <div class="doc" contenteditable="true">
+    This is initial call a widget needs to make to let Grist know that
+    it is ready to process incoming events.
+  </div>
+  <form>
+    <script type="code">
```

Comment:
Scrolling in this widget is awkward until you figure out the code boxes are capturing scroll events. Maybe they could capture scroll only if they have been clicked into?